packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeDetailsItem/MT5TradeDetailsItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, Fragment, useRef } from 'react';
+import { useHover } from 'usehooks-ts';
+import { qtMerge, Text, useBreakpoint } from '@deriv/quill-design';
+import { Clipboard, Tooltip } from '../../../../../components/Base';
+import EditIcon from '../../../../../public/images/ic-edit.svg';
+
+type TMT5TradeDetailsItemProps = {
+    className?: string;
+    label?: string;
+    value: string;
+    variant?: 'clipboard' | 'info' | 'password';
```

Comment:
There should be a handler for situations where there is no variant since it is optional.